pom.xml

```diff
@@ -47,6 +45,20 @@
               <transformationSet>
                 <dir>.</dir>
                 <includes>
+                  <include>indexing.rdf</include>
```

Comment:
@acoburn, I am not yet seeing the `rdfs2html.xsl` file. Have you yet to commit it or am I missing something?